docs/docs/aztec/concepts/smart_contracts/functions/inner_workings.md

```diff
@@ -73,7 +75,9 @@ Any state variables declared in the `Storage` struct can now be accessed as norm
 
 This function takes the application context, and converts it into the `PrivateCircuitPublicInputs` structure. This structure is then passed to the kernel circuit.
 
-## Unconstrained functions
```

Comment:
We currently don't have a `#[aztec(unconstrained)]` attribute and I'm not sure we will. These are just vanilla noir unconstrained functions.

Reply:
updated to just say unconstrained functions instead of #[aztec(unconstrained)]